test/parser.js

```diff
@@ -1,4 +1,25 @@
+var fs              = require('fs');
```

Comment:
to debug test/parser.js the variables declared in test/test.js were included in parser.js
(debugging is no possible via test.js, due to `eval(fs.readFileSync(__dirname+'/parser.js')+'');`)